README.md

```diff
@@ -6,53 +6,143 @@
+## Installing Tapioca in a project with no existing RBI files (i.e. a fresh Sorbet project)
 
-```shell
-$ bundle exec pry
-[1] pry(main)> require 'better_html'
-=> true
-[2] pry(main)> BetterHtml
-=> BetterHtml
-[3] pry(main)> BetterHtml::Parser
-NameError: uninitialized constant BetterHtml::Parser
-from (pry):3:in `__pry__`
-[4] pry(main)> require 'better_html/parser'
-=> true
-[5] pry(main)> BetterHtml::Parser
-=> BetterHtml::Parser
+### 1. Add this line to your application's `Gemfile`
+
+```ruby
+group :development do
+  gem 'tapioca', require: false
+end
 ```
+and run `bundle install`
 
-In order to make sure that `tapioca` can reflect on that type, we need to add the line `require "better_html/parser"` to the `sorbet/tapioca/require.rb` file. This will make sure `BetterHtml::Parser` is loaded into memory and a type annotation is generated for it in the `better_html.rbi` file. If this extra `require` line is not added to `sorbet/tapioca/require.rb` file, then the definition for that type will be missing from the RBI file.
+### 2, Initialize folder structure
 
-If you ever run into a case, where you add a gem or update the version of a gem and run `tapioca sync` but don't have some types you expect in the generated gem RBI files, you will need to make sure you have added the necessary requires to the `sorbet/tapioca/require.rb` file.
+Command: `tapioca init`
 
-You can use the command `tapioca require` to auto-populate the `sorbet/tapioca/require.rb` file with all the requires found
-in your application. Once the file generated, you should review it, remove all unnecessary requires and commit it.
+This will create the `sorbet/config` and `sorbet/tapioca/require.rb` files for you, if they don't exist. If any of the files already exist, they will not be changed.
 
-## How does tapioca compare to "srb rbi gems" ?
+### 3. Generate for gems
 
-[Please see the detailed answer on our wiki](https://github.com/Shopify/tapioca/wiki/How-does-tapioca-compare-to-%22srb-rbi-gems%22-%3F)
+Command: `tapioca generate [gems...]`
 
-## Installation
+This will generate RBIs for the specified gems and place them in the RBI directory.
 
-Add this line to your application's `Gemfile`:
+### 4. Generate for all gems in Gemfile
 
-```ruby
+Command: `tapioca sync`
+
+This will sync the RBIs with the gems in the Gemfile and will add, update, and remove RBIs as necessary.
+
+### 5. Generate the list of all unresolved constants
+
+Command: `tapioca todo`
+
+This will generate the file `sorbet/rbi/todo.rbi` defining all unresolved constants as empty modules.
+
+### 6. Generate DSL RBI files
+
+Command: `tapioca dsl [constant...]`
+
+This will generate DSL RBIs for specified constants (or for all handled constants, if a constant name is not supplied). You can read about DSL RBI generators supplied by `tapioca` in [the manual](manual/generators.md).
+
+## Installing Tapioca into a project that already uses Sorbet (and has RBI files)
```

Comment:
We should probably link to this page of the wiki: https://github.com/Shopify/tapioca/wiki/Migrating-to-Tapioca.